Multisig/UI/Transaction/ExecuteTransaction/SuccessViewController.swift

```diff
@@ -21,26 +21,29 @@ class SuccessViewController: UIViewController {
     private var bodyText: String?
     private var primaryAction: String?
     private var secondaryAction: String?
-    private var trackingEvent: TrackingEvent?
     private var wasNavBarHidden: Bool = false
 
-    var trackingParams: [String: Any]? = nil
+    private var trackingEvent: TrackingEvent?
+    private var trackingParams: [String: Any]? = nil
 
     var onDone: (_ didTapPrimary: Bool) -> Void = { _ in }
     
     convenience init(
         titleText: String?,
         bodyText: String?,
         primaryAction: String?,
-        secondaryAction: String?,
-        trackingEvent: TrackingEvent?
+        secondaryAction: String?
     ) {
         self.init(nibName: nil, bundle: nil)
         self.titleText = titleText
         self.bodyText = bodyText
         self.primaryAction = primaryAction
         self.secondaryAction = secondaryAction
+    }
+
+    func setTrackingData(trackingEvent: TrackingEvent?, trackingParams: [String: Any]? = nil) {
```

Comment:
Why this has been moved to new method, we can still keep it in the initializer and I would keep it there

Reply:
controller can be created without tracking info as it is optional and not essential. Also this way decision about tracking parameters can be delayed.

Reply:
additionally with this method event and it's parameters are bundled together now.

Reply:
I would actually still prefer to find a way to do all in initializer, better than two calls every time, as you can see in the places we currently use, but if you want to keep it like this then fine